lib/krane/deprecated_deploy_task.rb

```diff
@@ -453,9 +453,11 @@ def replace_or_create_resource(resource, force: false)
         ["replace", "-f", resource.file_path]
       end
 
-      _, _, status = kubectl.run(*args, log_failure: false)
+      _, _, status = kubectl.run(*args, log_failure: false, output_is_sensitive: resource.sensitive_template_content?,
+        raise_if_not_found: true)
       return [nil, status] if status.success?
```

Comment:
actually, now that we're doing the `create` inside the `rescue`, we _always_ want to return `[err, status]`; otherwise we return `nil` when the command fails with a non-404, which is not a valid return value for this method.